backend/cellguide/pipeline/canonical_marker_genes/utils.py

```diff
@@ -0,0 +1,91 @@
+import requests
```

Comment:
can you add some basic unit tests for the things in here? should be pretty easy to write and string manipulation is a common source of bugs